inbm/trtl/util/commandWrapper.go

```diff
@@ -10,23 +10,40 @@ import (
 
 // ExecCommandWrapper is an interface used for all docker commands
 type ExecCommandWrapper interface {
-	Run(string, string, []string) error
-	CombinedOutput(string, string, []string) ([]byte, error)
+	Run(string, string, []string, bool) error
+	CombinedOutput(string, string, []string, bool) ([]byte, error)
 }
 
 // ExecCommandWrap is the structure used with the ExeCommandWrapper interface
 type ExecCommandWrap struct{}
 
+const chroot = "/usr/sbin/chroot"
+const host = "/host"
+
 // Run makes actual call to exec.Command to run the command.
-func (ec ExecCommandWrap) Run(command string, directory string, args []string) error {
-	cmd := exec.Command(command, args...)
-	cmd.Dir = directory
-	return cmd.Run()
+// if chrootHost is true, run command in a chroot on /host
+func (ec ExecCommandWrap) Run(command string, directory string, args []string, chrootHost bool) error {
```

Comment:
Looking at this later on or for someone who isn't familiar, I'm not sure that it's clear that the chrootHost is needed for it to run in a container.  Is that the only reason for this?  If so, maybe explain a little better for someone else looking at the code.

Reply:
ah good point. this could be confusing down the line.